### DISCUSSION TITLE
How to run postinstallation commands in tauri for all 3 platform

### DISCUSSION THREAD

**Discussion by Pankaj-SinghR** [tauri-apps/tauri#7426] [Q&A]
Hi, i want to run few command before install (kind of postinstallation script) for tauri desktop application, how can i achieve this?
looking for all 3 platform

**Comment by FabianLars** [comment]
This is not implemented in Tauri itself right now and of the currently supported bundle types, only .deb, .dmg, .msi and setup.exe would theoritaically support this but of course no real javascript/rust support for these scripts when we use the "native" postinstall steps of these bundle types.
If you need js/rust support then it's up to the app itself either way at which point you could keep track of the installation step with a simple file (check if it exists, if not run postinstall stuff on first launch and create the file).

If the native postinstall thing is enough feel free to open a feature request :)
